edx_credentials_themes/templates/edx.org/credentials/programs/base.html

```diff
@@ -15,14 +16,15 @@
     <div class="accomplishment-metadata">
       <div class="accomplishment-metadata-bit accomplishment-metadata-aboutplatform">
         <h3 class="accomplishment-metadata-title">
-          {% trans "About edX" %}
+          {% trans "About edX" as tmsg %}{{ tmsg | htmlescape }}
         </h3>
         <p class="accomplishment-metadata-copy">
-          {% blocktrans trimmed %}
-            <a href="https://www.edx.org">edX</a> offers interactive online classes and MOOCs from
+          {% blocktrans trimmed asvar accomplishment_metadata_details %}
+            {link_start} edX{link_end} offers interactive online classes and MOOCs from
             the world’s best universities, including MIT, Harvard, Berkeley, University of Texas, and many others. edX
             is a non-profit online initiative created by founding partners Harvard and MIT.
           {% endblocktrans %}
+          {% interpolate_html accomplishment_metadata_details  link_start='<a href="https://www.edx.org">'|safe link_end='</a>'|safe %}
```

Comment:
Nit: I think there's an extra space between `accomplishment_metadata_details` and `link_start`.

Reply:
👍